pages/home/index.js

```diff
@@ -1,13 +1,13 @@
-import Background from "@/components/common/Background/beige.js";
-import Header from "@/components/common/Heading/Home.js";
-import DailyQuotes from "@/components/pages/home/DailyQuote.js";
+import BackgroundIllustration from "@/components/common/Background/Illustration.js";
+import Header from "@/components/common/Heading/HomePage.js";
+import DailyQuote from "@/components/pages/home/DailyQuote.js";
 
 export default function HomePage() {
   return (
     <>
-      <Background></Background>
+      <BackgroundIllustration color={"#b69b8a"}></BackgroundIllustration>
```

Comment:
Since you are not using children you can add the Component I think like this:
`<BackgroundIllustration color={"#b69b8a"} />`